librz/core/cmd/cmd_analysis.c

```diff
@@ -6528,3 +6350,100 @@ RZ_IPI RzCmdStatus rz_delete_global_imports_handler(RzCore *core, int argc, cons
 	rz_analysis_purge_imports(core->analysis);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_analysis_data_handler(RzCore *core, int argc, const char **argv) {
+	int count = argc > 1 ? rz_num_math(core->num, argv[1]) : 2 + (core->blocksize / 4);
+	if (count < 1) {
+		count = 1;
+	}
+	int depth = argc > 2 ? rz_num_math(core->num, argv[2]) : 1;
+	if (depth < 1) {
+		depth = 1;
+	}
+	int wordsize = argc > 3 ? rz_num_math(core->num, argv[3]) : 0;
+	rz_core_analysis_data(core, core->offset, count, depth, wordsize);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analysis_data_function_handler(RzCore *core, int argc, const char **argv) {
+	RzAnalysisFunction *fcn = rz_analysis_get_fcn_in(core->analysis, core->offset, RZ_ANALYSIS_FCN_TYPE_ANY);
+	if (!fcn) {
+		RZ_LOG_ERROR("Function not found at the 0x%08" PFMT64x " offset\n", core->offset);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	int i;
+	bool gap = false;
+	ut64 gap_addr = UT64_MAX;
+	ut32 fcn_size = rz_analysis_function_size_from_entry(fcn);
+	char *bitmap = calloc(1, fcn_size);
+	if (bitmap) {
+		RzAnalysisBlock *b;
+		RzListIter *iter;
+		rz_list_foreach (fcn->bbs, iter, b) {
+			int f = b->addr - fcn->addr;
+			int t = RZ_MIN(f + b->size, fcn_size);
+			if (f >= 0) {
+				while (f < t) {
+					bitmap[f++] = 1;
+				}
+			}
+		}
+	}
+	for (i = 0; i < fcn_size; i++) {
+		ut64 here = fcn->addr + i;
+		if (bitmap && bitmap[i]) {
+			if (gap) {
+				rz_cons_printf("Cd %" PFMT64u " @ 0x%08" PFMT64x "\n", here - gap_addr, gap_addr);
+				gap = false;
+			}
+			gap_addr = UT64_MAX;
+		} else {
+			if (!gap) {
+				gap = true;
+				gap_addr = here;
+			}
+		}
+	}
+	if (gap) {
+		rz_cons_printf("Cd %" PFMT64u " @ 0x%08" PFMT64x "\n", fcn->addr + fcn_size - gap_addr, gap_addr);
+	}
+	free(bitmap);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analysis_data_function_gaps_handler(RzCore *core, int argc, const char **argv) {
+	ut64 end = UT64_MAX;
+	RzAnalysisFunction *fcn;
+	RzListIter *iter;
+	int i, wordsize = (core->rasm->bits == 64) ? 8 : 4;
```

Comment:
```suggestion
	int i, wordsize = core->rasm->bits == 64 / 8;
```
? wdyt?

Reply:
ehm, what? i think you meant `core->rasm->bits / 8`

Reply:
OFC 😅